CHANGELOG.md

```diff
@@ -428,6 +449,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Removed State_Met%LWI and input meteorology LWI from carbon simulation run config files
 - Removed function `CLEANUP_UCX`; deallocations are now done in `state_chm_mod.F90`
 
+## [14.4.3] - 2024-08-13
```

Comment:
Why are these lines so far down? Perhaps a merge issue? I think they need to be moved up.

Reply:
Thanks Melissa, I can fix that in a post-merge fix.

Reply:
OK. I will approve then.